bitbucket-pipelines.yml

```diff
@@ -0,0 +1,23 @@
+image: gcc:10.2
+
+clone:
+  depth: full  # SonarQube scanner needs the full history to assign issues properly
+
+pipelines:
+  branches:
+    master:
+      - step:
+          name: Build and analyze on SonarQube
+          script:
+            - curl --create-dirs -sSLo $HOME/.sonar/build-wrapper-linux-x86.zip ${SONAR_HOST_URL}/static/cpp/build-wrapper-linux-x86.zip
+            - unzip -o $HOME/.sonar/build-wrapper-linux-x86.zip -d $HOME/.sonar/
+            - export PATH=$HOME/.sonar/build-wrapper-linux-x86:$PATH
+            - export BW_OUTPUT=bw-output
```

Comment:
Is there a way to move this out of the script into a yml field? Maybe `env:` or `variables:`?

Reply:
Also, to keep consistent with other examples and be explicit, I would spell `build_wrapper_output_directory` in full

Reply:
Unfortunately no, I did not find a way to move the variables in a different section